app/models/models.go

```diff
@@ -455,3 +455,21 @@ type PurgeQueueRequest struct {
 }
 
 func (r *PurgeQueueRequest) SetAttributesFromForm(values url.Values) {}
+
+/*** SNS */
```

Comment:
@Admiral-Piett How do you think about separating models/response files for SQS and SNS like `models_sqs.go` `models_sns.go`?
The usages may be the same with merged `models.go`, like `models.CreateQueueRequest` `models.CreateTopicRequest` .
The separation is only for maintainability.

Reply:
Hm, generally I am trying to keep things that are used the same way in single files.  This helps me avoid confusion when I'm looking for something shared.  I'm thinking about stuff like `BASE_RESPONSE_METADATA`.  

There are a thousand ways to organize it though, so I'm game to try to your `models_sqs/models_sns` split if you like.

Reply:
I'm sorry, I think we may have had a similar conversation before.

I understand your feelings, so let's move forward together without separating them for now 👍 
This is a topic we can revisit to once all API migrations are complete.